rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -334,6 +336,22 @@
           }
         }
       }
+    },
+    "blocked": {
+      "title": "Blocking",
+      "description": "A patron with a blocked account cannot request and borrow items, but can still renew and check in items.",
+      "type": "boolean"
+    },
+    "blocked_note": {
+      "title": "Reason",
+      "type": "string",
+      "description": "The reason is displayed in the circulation module and is visible by the patron in his account.",
```

Comment:
Is this description actually necessary for a librarian? Keep in mind that description are going to be translated.

Reply:
@iGormilhit Yes I think it is. This way, the librarian knows where the note will be displayed and, especially, that the note is displayed to the end user.

Reply:
@pronguen Ok, thanks.